packages/core/usePointerLock/demo.vue

```diff
@@ -0,0 +1,91 @@
+<script setup>
+import { ref, watch } from 'vue-demi'
+import { useMouse, usePointerLock } from '@vueuse/core'
+
+const { lock, unlock, element } = usePointerLock()
+const { x, y } = useMouse({ type: 'movement' })
+const rotY = ref(-45)
+const rotX = ref(0)
+
+watch([x, y], ([x, y]) => {
+  if (!element.value)
+    return
+  rotY.value += x / 2
+  rotX.value -= y / 2
+})
+</script>
+
+<template>
+  <div class="scene">
+    <div id="demo-cube" class="cube" @mousedown.capture="lock" @mouseup="unlock">
+      <div class="bases">
+        <span style="--i: 1" />
+        <span style="--i: -1" />
+      </div>
+      <div class="sides">
+        <span style="--i: 0" />
+        <span style="--i: 1" />
+        <span style="--i: 2" />
+        <span style="--i: 3" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<style scoped>
```

Comment:
We usually try to use Unocss for the styles in our demos. Can you check the other demos and follow how we do styles there?

Reply:
Where should I store custom rules and shortcuts? I think it's overkill to use root `unocss.config.ts` for styling a single demo. Can I configure it in the demo itself?

Reply:
Any custom ones you can keep as is